components/centraldashboard/public/components/main-page.js

```diff
@@ -273,7 +273,7 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
                 hideSidebar = true;
```

Comment:
changes in components `components/centraldashboard/public` came from re-running npm commands